### ISSUE TITLE
Test snippets should be compiled by the test suite

### ISSUE THREAD

**Issue by ryantrinkle** [reflex-frp/reflexfrp.org#4]
See https://github.com/reflex-frp/reflexfrp.org/pull/3#issuecomment-290739524

**Comment by dfordivam** [comment]
Sorry, which test suite are we referring to?
Are we talking about CI?

**Comment by ryantrinkle** [comment]
@dfordivam I don't think we have one yet!  But it would be great if we did, so that we could be reasonably sure that the examples don't get out of date.
